### DISCUSSION TITLE
Virtual text display (and keybinding to accept it) ?

### DISCUSSION THREAD

**Discussion by muzhancun** [ayamir/nvimdots#623] [Q&A] [answered]
### Feature description

Is there any way to display virtual text of copilot suggestion like [copilot-cmp](https://user-images.githubusercontent.com/32016110/173933674-9ad85a5a-5ad7-41cd-9fcc-f5a698cc88ae.png)? I can't find the config QAQ.

**Comment by charliie-dev** [comment]
This should be in discussions, transferred.

**Comment by charliie-dev** [comment]
I understand the virtual texts look cool in the first place, but once u start to use lots of completion from `cmp`, u will find the copilot vertual texts are super annoying. I know it b/c I was on the same path once, let the copilot suggestion lives in the `cmp` drop-down list is the best approach.

**Comment by charliie-dev** [comment, marked as answer]
If u really want to, here's the readme of both repos:
https://github.com/zbirenbaum/copilot-cmp
https://github.com/zbirenbaum/copilot.lua

What you're asking is basically redesigning the completion system of this nvimdots, there's no easy setup for that. I think u will need to finish it on your own.

**Reply by muzhancun** [reply]
OK, your config is really good

**Reply by charliie-dev** [reply]
It's actually not mine repo. It's ayamir's lol
